cli/src/commands/restore.rs

```diff
@@ -116,12 +116,12 @@ pub(crate) fn cmd_restore(
         return Err(user_error(
             "`jj restore` does not have a `--revision`/`-r` option. If you'd like to modify\nthe \
              *current* revision, use `--from`. If you'd like to modify a *different* \
-             revision,\nuse `--to` or `--changes-in`.",
+             revision,\nuse `--into` or `--changes-in`.",
```

Comment:
Thanks for not missing this instance. That reminds me that there might be docs to update too. Looks like only one instance in FAQ.md. Can you update that too? Thanks

Reply:
Indeed. I fixed the handful of matches of `rg 'restore.*--to'`, but apparently that was the only instance in the doc.